File: sysutils/rsfetch/files/patch-src_util.rs
--- src/util.rs.orig	2020-03-16 13:14:06 UTC
+++ src/util.rs
@@ -8,6 +8,7 @@ pub enum OS {
     FreeBSD,
     OpenBSD,
     NetBSD,
+    DragonflyBSD,
     Other
 }
 
@@ -23,11 +24,12 @@ impl OSInfo { 
             .output()?.stdout.iter()
             .for_each(|b| uname.push(*b as char));
         let os = match uname.replace("\n", "").trim().as_ref() {
-            "Linux"   => OS::Linux,
-            "FreeBSD" => OS::FreeBSD,
-            "NetBSD"  => OS::NetBSD,
-            "OpenBSD" => OS::OpenBSD,
-            &_        => OS::Other,
+            "Linux"     => OS::Linux,
+            "FreeBSD"   => OS::FreeBSD,
+            "NetBSD"    => OS::NetBSD,
+            "OpenBSD"   => OS::OpenBSD,
+            "DragonFly" => OS::DragonflyBSD,
+            &_          => OS::Other,
         };
 
         Ok(os)

File: dns/dnsmasq-devel/files/patch-src_util.c
--- src/util.c.orig	2024-12-20 21:57:21 UTC
+++ src/util.c
@@ -41,7 +41,7 @@ static int outleft = 0;
 static u32 out[8];
 static int outleft = 0;
 
-void rand_init()
+void rand_init(void)
 {
   int fd = open(RANDFILE, O_RDONLY);
   
